### DISCUSSION TITLE
Game problems.

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1321] [Help and Support] [answered]
When I delete a lot of objects in the game, everything freezes, a black screen and Windows 10 continues to work, all programs have closed (I noticed this when Chrome is turned on with a video), but I can hear the sound of a game error. Rebooted the PC 3 times and passed, an image appeared. New PC, AMD Athlon 200GE. I use the built-in video card which is in the processor. What's wrong?

**Comment by dc-redeclipse** [comment, marked as answer]
This indicated a typical hardware problem.
You should do a mem check and disk check first or your new cpu had a problem with your mainboard or the mainbaord or cpu is faulty.
This is by 100% sure a hardware issue and I am relatively sure its a memory problem.
To evaluate that you can also run benchmark on all your hardware type to prove this.
I can guarantee you that this behavior cant be produced by red eclipse.

**Comment by dc-redeclipse** [comment]
@KostyaPay did my advise solve your problem?
In that case we would close this issue.
